apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -1435,14 +1588,29 @@ static void tc_fs_vfs_rename(void)
 */
 static void tc_fs_vfs_ioctl(void)
 {
-	int fd, ret;
+	int fd;
+	int fd1;
```

Comment:
@kavyasrinunna 
Use fd1 and fd2, instead of fd and fd1

Reply:
done